pubsub/cloud-client/publisher.py

```diff
@@ -22,6 +22,7 @@
 """
 
 import argparse
+import concurrent.futures
```

Comment:
Should you explicitly add the futures backport to requirements.txt for Python 2.7 compatibility?

Reply:
Done; requirements.txt updated.